pkg/util/docker/event_pull.go

```diff
@@ -67,6 +68,7 @@ func (d *DockerUtil) processContainerEvent(msg events.Message) (*ContainerEvent,
 		}
 	}
 	if d.cfg.filter.computeIsExcluded(containerName, imageName) {
+		log.Debugf("events from %s are skipped as the image is excluded for the event collection", containerName)
```

Comment:
That's gonna be spammy. If you think that output can be useful for support, we can add it as a `trace` level log (lower than debug). IIRC we don't log filtered out containers for metrics retrieval

Reply:
I believe we should have trace of this at some point - I'll move it to log.Trace(), thanks for the pointer.